homeassistant/components/kraken/sensor.py

```diff
@@ -23,25 +24,26 @@
 async def async_setup_entry(hass, config_entry, async_add_entities):
     """Add kraken entities from a config_entry."""
 
-    @callback
     async def async_update_sensors(
```

Comment:
We can now make this a callback instead since we don't await inside anymore.
```suggestion
    @callback
    def async_update_sensors(
```